content/v2/zones.markdown

```diff
@@ -117,3 +117,44 @@ Responds with HTTP 200, renders the zone file.
 <%= pretty_print_fixture("/getZoneFile/success.http") %>
 ~~~
 
+
+## Get zone distribution {#get-zone-distribution}
+
+    GET /:account/zones/:zone
+
+Checks if a zone is fully distributed across DNSimple nodes.
```

Comment:
Do you think it's worth it to explain what `fully distributed` means (or linking a corresponding support article, if any)?

Reply:
I second that thought, looking at this from customers :eyes: it might be pretty confusing. As it implies some internal (special) knowledge.

Maybe rephrasing is enough, coming up with a full explanation might be overkill and unnecessary for most customers. something along the lines of:

```
Checks if a zone change is fully distributed to all our nameservers in all different regions we support.
```

Maybe link in there to the regions docs.